docs/pages/state-query/dozer.mdx

```diff
@@ -0,0 +1,573 @@
+import { CollapseCode } from "../../components/CollapseCode";
+import { Callout } from "nextra/components";
+
+# Dozer
+
+If there is a dozer instance serving a blockchain, as there is for [Redstone](https://redstone.xyz/) and [Garnet](https://garnetchain.com/docs/what-is-redstone), you can use it to run queries on the data of any `World` on that blockchain.
+
+The query language is a subset of [the SQL `SELECT` command](<https://en.wikipedia.org/wiki/Select_(SQL)>).
+
+## Example `World`
+
+On Garnet there is a `World` at address [`0x95F5d049B014114E2fEeB5d8d994358Ce4FFd06e`](https://explorer.garnetchain.com/address/0x95F5d049B014114E2fEeB5d8d994358Ce4FFd06e) that runs a slightly modified version of [the React template](https://github.com/latticexyz/mud/tree/main/templates/react).
+You can see the data schema for the `World` [in the block explorer](https://explorer.garnetchain.com/address/0x95F5d049B014114E2fEeB5d8d994358Ce4FFd06e?tab=mud).
+
+### Create a client to access the `World`
```

Comment:
should we move this section below the curl section given it's not needed for the curl section?

Reply:
I wasn't sure, because you can also use it to see what the curl results should be like. But once the explorer is out we can use that, so no need to worry about it.